static/app/components/teamSelector.tsx

```diff
@@ -115,15 +135,19 @@ type TeamOption = GeneralSelectValue & {
 };
 
 function TeamSelector(props: Props) {
-  const {includeUnassigned, styles, ...extraProps} = props;
-  const {teamFilter, organization, project, multiple, value, useId, onChange} = props;
+  const {allowCreate, includeUnassigned, styles, onChange, ...extraProps} = props;
+  const {teamFilter, organization, project, multiple, value, useId} = props;
 
   const api = useApi();
   const {teams, fetching, onSearch} = useTeams();
 
   // TODO(ts) This type could be improved when react-select types are better.
   const selectRef = useRef<any>(null);
 
+  const canCreateTeam = organization.access.includes('project:admin');
+  const canAddTeam = organization.access.includes('project:write');
+  const includeCreate = allowCreate && canCreateTeam;
```

Comment:
shall we check for `allowCreate` here too?

Reply:
I do 🙃

Reply:
oh yes! 😄